packages/core/test/rules/implement_methods.ts

```diff
@@ -523,4 +523,73 @@ ENDCLASS.`;
     expect(issues[0]?.getMessage()).to.equals(undefined);
   });
 
+  it("missing method of unknown aliases", async () => {
+    const issues = await runMulti([
+      {
+        filename: "cl_foo.clas.abap",
+        contents: `CLASS lcl DEFINITION.
+    PUBLIC SECTION.
+      ALIASES blah FOR zlif~blah.
+  ENDCLASS.
+  
+  CLASS lcl IMPLEMENTATION.
+  ENDCLASS.`,
+      },
+    ]);
+    expect(issues.length).to.equals(1);
```

Comment:
hmm, should this be an error? `blah` might not be a method

Reply:
That's what I meant above. BTW, I thought I might have to check parent classes too to get this right